dbt/include/duckdb/macros/utils/upstream.sql

```diff
@@ -0,0 +1,25 @@
+{%- macro register_external_upstream() -%}
+{% if execute %}
+  {% for node in model['depends_on']['nodes'] %}
+    {% set upstream = graph.nodes.get(node) %}
+    {% if upstream %}
+      {%- set upstream_rel = api.Relation.create(
+        database=upstream['database'],
+        schema=upstream['schema'],
+        identifier=upstream['alias']
```

Comment:
`alias` here is intentional? (I'll admit the distinction between `identifier` and `alias` in dbt isn't always clear to me)

Reply:
In my understanding, the `identifier` is the property of the dbt Relation (encapsulating database details), where `alias` is a property of the dbt node. The `alias` (not the node `name`) is mapped to `identifier` when a node is parsed into a Relation object:
https://github.com/dbt-labs/dbt-core/blob/eb200b46877c31469f410b7d47dfd245473ed7cb/core/dbt/adapters/base/relation.py#L226-L245